### ISSUE TITLE
the Web UI can not show completly

### ISSUE THREAD

**Issue by JOJOJOFran** [hashicorp/consul#4136]
1.10 (win  64bit)& 1.06(win 32bit&win 64bit) can not show completly ,I've tried all of these versions 

![image](https://user-images.githubusercontent.com/19874689/40177986-c382b788-5a12-11e8-84fe-fb5de087ca9c.png)

![1dimgbfk79sra md 3drx 5](https://user-images.githubusercontent.com/19874689/40183644-edca89c2-5a20-11e8-9a40-4406476e21e6.png)

can you give me some advice

**Comment by MagnumOpus21** [comment]
Hi @JOJOJOFran. I did a fresh install of Consul on Windows 10.

I followed these steps:

1. Launched a Consul agent using `consul agent -dev`
2. Accessed the web UI in a browser at the default address (`http://localhost:8500`)

The UI loaded correctly for me and I wasn't able to reproduce the issue with the UI that you faced.

It would be helpful if you could provide:

- Your OS (build/version)
- Services (if any were configured to be used).
- Steps taken to launch Consul and the UI.
- Whether you had enabled the new UI, using the environmental variable and then reverted back to using the old UI again.

**Comment by JOJOJOFran** [comment]
Hi,@MagnumOpus21 ,Thank for you  #reply
I tried to install the Consul on four different PC (Win10 2,Win7 1,Win8 1) followed these steps(I also tried other steps):
1.put the exe into the path C://windows/system32
2.using the cmd : consul agent -dev
3.Accessed the localhost:8500

And I’ve got the result: 
Success: Win10(Enterprise 10.0.16299),  Win7
Fail:Win10(my Win10), Win8.1(Enterprise 6.3.9600)

I will add the version of my Win10 lately

**Comment by MagnumOpus21** [comment]
Hey @JOJOJOFran . I installed Consul on all the versions of Windows that you have mentioned. I did not face any issue in accessing the Consul UI. I tried accessing Consul on the following browsers:

- Internet Explorer (Out of the box version)
- Microsoft Edge 
- Firefox 
- Chrome

All the browsers were updated to their latest versions. I'm closing the issue down, as I could not reproduce your issue. :(  If I had missed anything feel free to comment here.
